pages/400.html

```diff
@@ -1,9 +1,13 @@
 
 <html>
     <head>
-        <link type="text/css" rel="stylesheet" media="all" href="error_style/style.css">
-        <link type="text/css" rel="stylesheet" media="all" href="error_style/bootstrap_style.css">
-        <link type="text/css" rel="stylesheet" media="all" href="error_style/additional_style.css">
+        <!--
+        For production environments, it's necessary to have the href start with /errors.
+        To do development, change to error_style/<filename>.css
```

Comment:
Could we add a directory containing a symlink so `errors/error_style` -> `error_style`?